frontend/actions/user.ts

```diff
@@ -1,30 +1,46 @@
+import qs from 'querystring'
 import useSWR from 'swr'
 
-import { User } from '../types/types'
 import { ErrorInfo, fetcher } from '../utils/fetcher'
 
-export function useListUsers() {
+interface EntityKind {
```

Comment:
Does `EntityGroup` or `EntityResults` make more sense as a name? The entity kind is the string, so it feels strange to also refer to the whole object as that.